src/key_transformations.scad

```diff
@@ -34,17 +34,21 @@ module rotated() {
   children();
 }
 
-module stabilized(mm=12, vertical = false, type="cherry") {
+module stabilized(mm=12, vertical = false, type=undef) {
```

Comment:
pretty frustrating to have your stabilizer preferences overwritten by `lshift()`